README.md

```diff
@@ -180,30 +159,20 @@ The OM package imported contains a new model for the UX apps. Non-admin users ne
 5.  In the "Model Access" panel on the right, select `Read,Update,Create and Delete`.
 6.  Click Save & Close.
 
-#### L) TEST THE WORKPLACE SERVICES APPS.
+#### M) TEST THE WORKPLACE SERVICES APPS.
 
 It's time to test with the Assistant Chat UI available from the Workplace Services app.  Make sure the user you are using has a primary location set, isn't the system or assistant user, and has the security group modified or created in step K.  If all edits were done correctly, you should see a chat icon appear at the bottom right of the Workplace Services apps.  If you know a room name, then try out the service request functionality by typing "the ________ room has a broken chair" and if you have reserve functionality, try "book a room".
```

Comment:
A screenshot of the Web Component and where it would show up would be good.  At least maybe mention the workspace service apps the assistant shows up in as it wont show up in all of them.

Reply:
added screenshot